captainhook.json

```diff
@@ -23,8 +23,12 @@
     ]
   },
   "pre-push": {
-    "enabled": false,
-    "actions": []
+    "enabled": true,
+    "actions": [
+      {
+        "action": "bin/hooks/pre-push '{$TARGET}' '{$URL}'"
```

Comment:
Diese Zeile funktioniert nach meinen Tests nicht richtig.
Neben den beiden `TARGET` und `URL` Parametern liest der Hook zusätzlich Eingabedaten aus der Standardeingabe (STDIN). Dieser Teil funktioniert damit nicht.
Ich hatte es ähnlich getestet, indem ich in der Action den pre-push von git-lfs ausführe:
```
git lfs pre-push '{$TARGET}' '{$URL}'
```
TARGET und URL wurden korrekt erkannt, aber die Werte aus STDIN wurden nicht richtig verarbeitet. Das führte dazu, dass die von git lfs getrackten Dateien nicht richtig über die Git LFS API gepusht werden. Das Pushen hat dann zwar funktioniert, aber die Datei wird in Github nicht mit Git LFS getrackt und beim auschecken gab es beim Auflösen der Pinter Fehler, dass die Datei nicht gefunden werden konnte.

Ich wollte noch einmal testen, ob man nicht eine eigene PHP-Klasse schreiben könnte, in der man alle benötigten Werte auslesen kann und dann den `git lfs pre-push`-Befehl ausführen kann.

Reply:
Ich habe dazu ein Issue im Captainhook Repo erstellt: https://github.com/captainhookphp/captainhook/issues/256